css/css-color/rec2020-004-ref.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Color 4: CSS Color 4: rec2020</title>
+<style>
+    .test { background-color: lab(85.7729% -160.7259 109.2319); width: 12em; height: 12em; } /* color(rec2020 0 1 0) converted to Lab */
```

Comment:
I'm getting the same conversion results to and from rec2020 if all components are either 0 or 1. So our conversions differ in the transfer function (which is pinned, 0->0 and 1->1, it's the part in the middle that is affected).